### PR TITLE
jaxws failure with GF 6.1 and JDK 11 fix.

### PR DESCRIPTION
**Pull request by gurunrao**
Signed-off-by:  gurunandan.rao@oracle.com

PR fixes JAXWS 852 failures(https://github.com/eclipse-ee4j/jakartaee-tck/wiki/Jakarta-EE-9.1-TCK-results).
Further details can be found in issues:
#649
#617
CI run with GF 6.1 + JDK 11 + PR - https://ci.eclipse.org/jakartaee-tck/job/guru/job/jakartaee-tck-guru/job/jaxws-sttck-fix/28/

### CODE REVIEW

File: src/com/sun/ts/tests/jaxws/api/jakarta_xml_ws/AsyncHandler/Client.java [hunk 1]
```diff
@@ -20,25 +20,30 @@
 
 package com.sun.ts.tests.jaxws.api.jakarta_xml_ws.AsyncHandler;
 
-import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Properties;
+import com.sun.ts.lib.util.*;
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jaxws/api/jakarta_xml_ws/AsyncHandler/Client.java:23]: +1, thanks for correcting these!

File: src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001.wsdl [hunk 2]
```diff
@@ -1,25 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!--
-
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
-
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001.wsdl:5]: I noticed that we are deleting the copyright notice, should that be kept?

File: src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001_schema1.xsd [hunk 3]
```diff
@@ -1,22 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!--
-
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
-
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001_schema1.xsd:5]: Should we keep the copyright notice?

File: src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23002.wsdl [hunk 4]
```diff
@@ -1,25 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!--
-
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
-
-    This program and the accompanying materials are made available under the
-    terms of the Eclipse Public License v. 2.0, which is available at
```
Comment (scottmarlow) [inline src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23002.wsdl:7]: Should we keep the copyright notice?

**Comment by scottmarlow** [comment]
Other than my question about removing the copyright from some files, this change looks good to me.

File: src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001.wsdl (hunk 2 above)
Comment (gurunrao) [reply src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001.wsdl:5]: Have added back the copyright, thanks for letting me know.

**Review by scottmarlow — APPROVED** [review]

**Pull request merged**